pyproject.toml

```diff
@@ -34,7 +34,7 @@ dummy-variable-rgx = "^(_+|(_+[a-zA-Z0-9_]*[a-zA-Z0-9]+?))$"
 [tool.pytest.ini_options]
 norecursedirs = [".cache", "__pycache__"]
 testpaths = "tests/"
-addopts = "-rsxX --showlocals --tb=native -p no:cacheprovider --import-mode=importlib"
```

Comment:
pytest wanted to switch to this import mode, but there are too many problems with it, so they're no longer planning to switch to this as the default and aren't encouraging people to use it.

In our case, it breaks pytest assert rewriting in some cases which makes some tests harder to debug.

Removing this flag returns us to `prepend` which manipulates the `sys.path`.